pages/api/[...mockApiRoute].ts

```diff
@@ -54,12 +57,14 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 
   if (req.method == 'GET') {
     if (
-      mockApiRoute === 'sites/b97f68cd-50e0-49bb-a850-108d4a9f7b7e/pv_actual'
+      mockApiRoute === 'sites/725a8670-d012-474d-b901-1179f43e7182/pv_actual'
     ) {
+      fakeDates(pvActualJson as any);
       res.status(200).json(pvActualJson);
     } else if (
-      mockApiRoute === 'sites/b97f68cd-50e0-49bb-a850-108d4a9f7b7e/pv_forecast'
+      mockApiRoute === 'sites/725a8670-d012-474d-b901-1179f43e7182/pv_forecast'
```

Comment:
oh yay now we won't have to change around our siteUUID between testing and deployment